docs/specs/metadata.yml

```diff
@@ -114,19 +114,33 @@ outputs:
 inputs:
   docfx.yml: |
     documentId:
-      sourceBasePath: sccm
-    name: sccm
```

Comment:
don''t change this test, it's from real world, for specific purpose. Add a new test?

Reply:
from the title, this test case is for the same purpose.
and spec test doesn't require real world case, right? more simple, more readable

---
In reply to: [320657688](https://github.com/dotnet/docfx/pull/5047#discussion_r320657688) [](ancestors = 320657688)

Reply:
OK, then how do you know the document id of X is really from A?

---
In reply to: [321588571](https://github.com/dotnet/docfx/pull/5047#discussion_r321588571) [](ancestors = 321588571,320657688)

Reply:
manually verified😆